src/serac/physics/heat_transfer.hpp

```diff
@@ -272,9 +272,6 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
 
   /**
    * @brief Non virtual method to reset thermal states to zero.  This does not reset design parameters or shape.
-   *
-   * @param[in] cycle The simulation cycle (i.e. timestep iteration) to intialize the physics module to
-   * @param[in] time The simulation time to initialize the physics module to
    */
```

Comment:
Is the doxygen ci test not working? Shouldn't this have caused a warning failure...

Reply:
im guessing with the new doxygen version it now counts as a warning. this is from the older failing job: https://dev.azure.com/llnl-serac/serac/_build/results?buildId=11561&view=logs&j=1be3ff84-5b97-5584-c1a4-7ebbbeb3b802&t=d32b9d99-ea3e-5518-9381-52c611e33fea&l=2000